index.html

```diff
@@ -359,12 +359,38 @@ <h3>
       additional members to control audio packetization.
     </p>
     <pre class="idl">partial dictionary RTCRtpEncodingParameters {
-  unsigned long ptime;
-  boolean adaptivePtime = false;
+    RTCResolutionRestriction restrictedResolution;
+    unsigned long            ptime;
+    boolean                  adaptivePtime = false;
 };</pre>
     <section id="rtcrtpencodingparameters-attributes">
       <h2>Dictionary {{RTCRtpEncodingParameters}} Members</h2>
       <dl data-link-for="RTCRtpEncodingParameters" data-dfn-for="RTCRtpEncodingParameters" class="dictionary-members">
+        <dt>
+          <dfn data-idl>restrictedResolution</dfn> of type <span class="idlMemberType">{{RTCResolutionRestriction}}</span>
+        </dt>
+        <dd>
+          <p>The resolution at which to restrict this encoding.</p>
+          <p>The default "scale resolution down by" factors (e.g. 4:2:1) are not
+          applied when this parameter is used. Instead, frames are either sent
+          as-is or they are downscaled by the encoder to uphold the
+          restrictions. The encoder will never upscale a frame. In simulcast,
+          this can result in dropping top layer(s) if the input frame is too
```

Comment:
Define "top layer". We should either send the first encoding in the list for which the restriction results in an unscaled frame or the last encoding in the list for which the restriction results in an unscaled frame, and not send the frame on the rest of the encodings.

Reply:
Updated this sentence, is this better?